exaudfclient/base/run_local.sh

```diff
@@ -1,3 +1,6 @@
+#!/bin/bash
+
+#shellcheck disable=SC1091
```

Comment:
.env file does not exist in repo. Need to disable 
See https://github-wiki-see.page/m/koalaman/shellcheck/wiki/SC1090

Reply:
ok

Reply:
Maybe add a comment in the code

Reply:
Done